app/views/admin/index.html.erb

```diff
@@ -48,6 +48,7 @@
       <li><%= link_to "Weather Stations", admin_weather_stations_path %></li>
       <li><%= link_to "Data Sources", admin_data_sources_path %></li>
       <li><%= link_to "Procurement Routes", admin_procurement_routes_path %></li>
+      <li><%= link_to "PROB data report", admin_prob_data_reports_path %></li>
```

Comment:
Can you add this to `/admin/reports/index.html.erb` instead of here, under the "Data & Tariffs" section?

Reply:
:+1: done